app/controllers/application_controller.rb

```diff
@@ -159,7 +159,6 @@ def space_repository
   end
 
   def handle_unauthorized(exception)
-    Sentry.capture_exception(exception, level: :warn)
```

Comment:
Ah, yeah, we don't need this anymore. I forgot that we were sending to Sentry before showing 404.